src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentsResource.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentsResource : ResourceBase, ITrustlyPaymentsResource
+    {
+        public TrustlyPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<TrustlyPayment> Create(TrustlyPaymentRequest paymentRequest)
```

Comment:
Codacy found an issue: [Make 'Create' a static method.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)

Reply:
@asbjornu this method can't be static either